docs/translations.rst

```diff
@@ -6,6 +6,38 @@ localization strings. For more comprehensive information, consult the
 `Django Docs translations page <https://docs.djangoproject.com/en/dev/topics/i18n/translation/>`_
 or `this helpful blog post <http://blog.bessas.me/post/65775299341/using-gettext-in-django>`_.
 
+How translations work
+---------------------
+There are three major components to the translations system - identifying
+strings that need to be translated, having human translators write translations
+for those strings, and then actually inserting those translations when
+appropriate.
+
+1. ``./manage.py makemessages`` goes over the source code and pulls out strings
+   that have been tagged for translation. It does not actually execute code, so
+   you really can't do anything fancy in these strings.
```

Comment:
It might be nice to specify what practices are considered "fancy". An example or two could be helpful, if there's a concise way to describe this, that would be even better.

Reply:
good point, I'm not really sure.  I think at its core you can really only put strings in here.  Mabe I should rephrase this like
```suggestion
   you can only use simple string literals here.
```
That is, formatting a string won't work, nor will referencing a variable that holds a string.  For example:
```python
MYSTR = "foo"
translated = gettext_noop(MYSTR)
```
This won't actually add any strings to the translation file, because to do so would require evaluating the code.

Reply:
Yeah, I actually totally missed this point in my mind, I was already limiting the scenario where a developer puts an f-string or call "".format() inside the gettext call.